README.md

```diff
@@ -13,18 +12,18 @@ The model was trained on raw T2-weighted images of SCI patients from seven sites
 
 ## Updates
 
-### 2024-XX-XX
+### 2024-07-24
 
-* We have released **SCIsegV2: A Universal Model for Intramedually Lesion Segmentation in SCI**. The new model is trained on a larger cohort covering both traumatic and non-traumatic SCI lesions. The new model is available for download using the same installation instructions as below.
-* The computation of midsagittal tissue bridges is now fully-automatied and powered by SCIsegV2. 
+* We have released **SCIsegV2: A Universal Model for Intramedullary Lesion Segmentation in SCI**. The new model is trained on a larger cohort covering both traumatic and non-traumatic SCI lesions. SCIsegV2 is available as part of the SCT via the `sct_deepseg` function; see the installation instructions below.
+* The computation of midsagittal tissue bridges is now fully-automated and powered by SCIsegV2. The automatic computation of tissue bridges is available via the `sct_analyze_lesion` function as part of SCT v6.4 and higher.
```

Comment:
I'm thinking whether we want to add Fig2 from https://arxiv.org/pdf/2407.17265 here to illustrate the tissue bridges.

Reply:
sure! I don't have the figure so I couldn't add it

Reply:
Okay! The figure added in https://github.com/ivadomed/model_seg_sci/commit/ec09d39dedadf4cd4a0295d5d3f08b45f533fe6b